packages/playback-core/src/types.ts

```diff
@@ -1,5 +1,3 @@
-/* eslint @typescript-eslint/triple-slash-reference: "off" */
```

Comment:
Per mux embed discussion/typing usage, you should be able to use this PR draft (https://github.com/muxinc/elements/pull/809/files#diff-0550a08b3e80a37aa663c63c5e486747f32d7ddea4e7b52eaa818bec25d044b6R1-R2) as a reference point for how to use the officially published typings (though several other things have changed since it was implemented)